db/structure.sql

```diff
@@ -28,22 +28,22 @@ CREATE TABLE `bmap_flowcell` (
   `recorded_at` datetime NOT NULL COMMENT 'Timestamp of warehouse update',
   `id_sample_tmp` int unsigned NOT NULL COMMENT 'Sample id, see "sample.id_sample_tmp"',
   `id_study_tmp` int unsigned NOT NULL COMMENT 'Study id, see "study.id_study_tmp"',
-  `experiment_name` varchar(255) COLLATE utf8_unicode_ci NOT NULL COMMENT 'The name of the experiment, eg. The lims generated run id',
-  `instrument_name` varchar(255) COLLATE utf8_unicode_ci NOT NULL COMMENT 'The name of the instrument on which the sample was run',
-  `enzyme_name` varchar(255) COLLATE utf8_unicode_ci NOT NULL COMMENT 'The name of the recognition enzyme used',
-  `chip_barcode` varchar(255) COLLATE utf8_unicode_ci NOT NULL COMMENT 'Manufacturer chip identifier',
-  `chip_serialnumber` varchar(16) COLLATE utf8_unicode_ci DEFAULT NULL COMMENT 'Manufacturer chip identifier',
+  `experiment_name` varchar(255) CHARACTER SET utf8 COLLATE utf8_unicode_ci NOT NULL COMMENT 'The name of the experiment, eg. The lims generated run id',
```

Comment:
@andrewsparkes hey, all these extra SQL changes came in in your latest commit - guessing we should get rid of them again?